src/heatmap-layer.ts

```diff
@@ -80,17 +80,23 @@ export class GeoArrowHeatmapLayer<
   renderLayers(): Layer<{}> | LayersList | null {
     const { data: table } = this.props;
 
-    const pointVector = getGeometryVector(table, EXTENSION_NAME.POINT);
-    if (pointVector !== null) {
-      return this._renderLayersPoint(pointVector);
-    }
+    if (this.props.getPosition !== undefined) {
```

Comment:
These changes don't look necessary for upgrading to deck v9? Or were these lines changed upstream and you're copying upstream into here?

Reply:
oh I didn't realize this PR was into the kyle/deck-v9 branch, instead of main. That's why I'm confused.

Reply:
And here I was thinking that would be the _less_ confusing approach :joy: